uk.co.compucorp.civicrm.tasksassignments/tests/phpunit/api/v3/DocumentTest.php

```diff
@@ -305,12 +303,13 @@ public function testDocumentCloningOnlyClonesCurrentRevision() {
   }
 
   public function testOnlyDocumentsAssignedToACaseCreateRevisionsOnUpdate() {
+    $originalCount = civicrm_api3('Activity', 'getcount');
```

Comment:
Instead of this, it's better to cleanup things in the `setUp` method, as it was done before

Reply:
I'm not sure I agree,  it could be possible that installing an extension will create an Activity. It's outside the scope of the test. I wouldn't put it in `setUp` anyway, since it is run for every test and a clean activity table is only required for this test.

Reply:
Well, the previous version of this class would delete all the activities (and a few more things) before running the tests, so my suggestion is just to keep the previous behavior.

I do think that the setUp method is the best place to add this and the reason is to remove the test everything that is not related to what is being tested. It's not always possible to do that, but here's I think it's quite simple.

Reply:
Instead of using original count I'm using a trait to cleanup the tables before the test. I still don't think this should be done on each test run because it's not relevant to most of the other tests.